src/main/java/com/basho/riak/client/core/query/timeseries/FullColumnDescription.java

```diff
@@ -0,0 +1,180 @@
+package com.basho.riak.client.core.query.timeseries;
+
+/**
+ * Holds a complete definition for a Table Column in Time Series Riak.
+ * Immutable once created.
+ *
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.4
+ */
+public class FullColumnDescription
+{
+    private final ColumnDescription baseColumnDescription;
```

Comment:
It might be more logical to use inheritance  instead of aggregation:

class FullColumnDescription extends ColumnDescription